research/delf/delf/python/training/README.md

```diff
@@ -197,11 +227,20 @@ Assuming the training output, the TensorFlow checkpoint, is in the
 This should be used when you are only interested in having a local feature
 model.
 
+```
+python3 model/export_local_model.py \
+  --ckpt_path=gldv2_training/delf_weights \
+  --export_path=gldv2_model_local
+```
+
+If the model was trained with the autoencoder described in the DELG paper, use
+the following export command instead:
+
 ```
 python3 model/export_local_model.py \
   --ckpt_path=gldv2_training/delf_weights \
   --export_path=gldv2_model_local \
-  --block3_strides
```

Comment:
I agree it's a good idea to remove `block3_strides`, since it's anyway True by default in both train.py and export_local_model.py

Reply:
If I understand correctly no change is require here, right?